modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/TextInputControlSkinTest.java

```diff
@@ -92,6 +94,17 @@
         passwordField.setText(null);
     }
 
+    @Test public void noNullPointerIfTextInputNotInScene() {
+        TextField textField = new TextField();
+        TextFieldSkin skin = new TextFieldSkin(textField);
+        textField.setSkin(skin);
+
+        // Check that no NullPointerException is thrown if the TextField is not in scene
+        // and that the default point is returned.
+        Point2D point = textField.getInputMethodRequests().getTextLocation(0);
+        assertEquals(new Point2D(0,0), point);
```

Comment:
Very minor : Please add a space between `0,0`

Reply:
I added the space and checked the fix again with the sample program. 
The fix works, the NullPointer does not occur anymore.